### PR TITLE
Add more binding for pull requests:

### PR DESCRIPTION
**Pull request by henryju**
* ability to list files of a pull request
* ability to add/remove/update PR review comment

### CODE REVIEW

**Comment by kohsuke** [comment]
Thanks for the pull rueqest. Unfortunately it looks like your IDE made a bunch of whitespace only changes that make it difficult for me to see the changes. I can plow through that with "git diff -w" but I'll have to come back to this later.

File: src/main/java/org/kohsuke/github/GHPullRequestFileDetail.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
+/*
+ * The MIT License
+ * 
+ * Copyright (c) 2013, Luca Milanesio
```
Comment (KostyaSha) [inline src/main/java/org/kohsuke/github/GHPullRequestFileDetail.java]: 2015

**Comment by KostyaSha** [comment]
"ability to add/remove/update PR review comment"
Where comments are stored? In PR or Issue?

**Comment by henryju** [comment]
This is comments you can made "inline" in the pull request (https://developer.github.com/v3/pulls/comments/)

**Comment by henryju** [comment]
@kohsuke sorry for the unexpected reformat. Tell me if you want that I fix the P/R.

**Comment by kohsuke** [comment]
Merge manually performed in 63136f64b74c9492050ea0a4256d98a8f9e05966.

**Pull request closed without merging**